src/main/java/com/nutomic/syncthingandroid/fragments/SettingsFragment.java

```diff
@@ -21,10 +24,14 @@
 
 import com.nutomic.syncthingandroid.R;
 import com.nutomic.syncthingandroid.activities.SyncthingActivity;
+import com.nutomic.syncthingandroid.preferences.WifiSsidPreference;
 import com.nutomic.syncthingandroid.syncthing.RestApi;
 import com.nutomic.syncthingandroid.syncthing.SyncthingService;
 
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
```

Comment:
I used the (cheaper) `HashSet` as default value, and the `TreeSet` is required for ordering of the SSID names in the summary. Can use `TreeSet` for default value as well if you mind the import. Since `TreeSet` is special (it requires comparable entries) I tend to avoid it when not explicitly required.